build.gradle

```diff
@@ -2,7 +2,12 @@ plugins {
 	id "java"
 	id "java-library"
 	id "maven-publish"
-	id "net.minecrell.licenser" version "0.4.1"
+	id "org.cadixdev.licenser" version "0.6.1"
```

Comment:
We are now using spotless, I believe.

Reply:
Hmm, better suited for another PR I believe